modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -683,6 +724,7 @@ protected Map<String, Object> prepareProps(Map<String, Object> props) {
 		props.put("showInfoButton", isShowInfoButton());
 		props.put("showResultsBar", isShowResultsBar());
 		props.put("showSearch", isShowSearch());
+		props.put("showSearchResultsTitle", isShowSearchResultsTitle());
```

Comment:
Can we put this props under FF, along with the other one?

Reply:
sure!